Project Walking/Source code/app/src/main/java/com/teamapricot/projectwalking/Reminder.java

```diff
@@ -0,0 +1,34 @@
+package com.teamapricot.projectwalking;
+
+import android.content.Context;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+public class Reminder {

```

Comment:
Add JavaDoc comment to the class, include @author and @version here. Look at the other classes for examples.